code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -212,6 +212,12 @@
 	. = ..()
 	if(methods & TOUCH)
 		exposed_mob.extinguish_mob() // extinguish removes all fire stacks
+	if(methods & VAPOR)
+		if(!isfelinid(exposed_mob))
+			return
+		exposed_mob.stun(1) //cancels any do_after
```

Comment:
am i crazy or are you looking for `Stun()`?

Reply:
Oh no